jekyll/_cci2/configuration-reference.md

```diff
@@ -1683,6 +1683,7 @@ includes:
 | or                                                                                                  | N logic statements | any argument is truthy                 | `or: [ false, true, false ]`                                         |
 | not                                                                                                 | 1 logic statement  | the argument is not truthy             | `not: true`                                                          |
 | equal                                                                                               | N values           | all arguments evaluate to equal values | `equal: [ 42, << pipeline.number >>]`{: class="table table-striped"} |
+{: class="table table-striped"}
```

Comment:
Should the class declaration be removed from the previous line?

Reply:
Ah so that's where it went when it got formatted. Thanks, fixed!